go.mod

```diff
@@ -213,3 +213,5 @@ replace google.golang.org/protobuf => github.com/lightninglabs/protobuf-go-hex-d
 go 1.22.6
 
 retract v0.0.2
+
+replace github.com/lightningnetwork/lnd/fn/v2 => github.com/ellemouton/lnd/fn/v2 v2.0.0-20241210050749-8c19ec06a189
```

Comment:
FWIW in https://github.com/lightningnetwork/lnd/pull/8512 I've made the change to update to `v2` just need to get CI grteen there (check commits failing IIRC) and it should be g2g there.